multi/imxrt-multi/imxrt-multi.c

```diff
@@ -258,125 +270,138 @@ static int createDevFiles(void)
 		return -1;
 	}
 
+	/* Pseudo devices */
+
+	if (mkFile(&dir, id_pseudoNull, "null", multi_port, MODE_DEFAULT) < 0) {
+		return -1;
+	}
+
+	if (mkFile(&dir, id_pseudoZero, "zero", multi_port, MODE_DEFAULT) < 0) {
+		return -1;
+	}
+
+	if (mkFile(&dir, id_pseudoFull, "full", multi_port, MODE_DEFAULT) < 0) {
+		return -1;
+	}
```

Comment:
Why isn't pseudodev lib optional? We might want to use posixsrv instead

Reply:
Done.